src/test/java/hudson/plugins/ec2/AmazonEC2CloudTest.java

```diff
@@ -32,6 +32,7 @@
 import com.cloudbees.plugins.credentials.CredentialsStore;
 import com.cloudbees.plugins.credentials.SystemCredentialsProvider;
 import com.cloudbees.plugins.credentials.domains.Domain;
+import org.htmlunit.FailingHttpStatusCodeException;
```

Comment:
```suggestion
```

Leftover from previous commits now removed I guess

Reply:
removed in 60e1b46a4ae4e2363216adf5e604acfa294512e7